packages/next/server/config-schema.ts

```diff
@@ -253,7 +253,14 @@ const configSchema = {
           type: 'boolean',
         },
         esmExternals: {
-          type: 'boolean',
+          oneOf: [
+            {
+              type: 'boolean',
+            },
+            {
+              const: 'loose',
+            },
+          ] as any,
```

Comment:
I saw `as any` elsewhere in this file wherever `oneOf?:` is used in the schema, which should probably be fixed for type safety, but I'm not sure why it breaks without that.